app.js

```diff
@@ -78,9 +87,59 @@ const STREAM_WORST_QUALITY_QUERY =
   '[data-a-target="player-settings-menu"] .tw-pd-05:last-child .tw-radio';
 const CHANNEL_STATUS_QUERY = ".tw-channel-status-text-indicator";
 const NO_DROPS_QUERY = 'div[data-test-selector="drops-list__no-drops-default"]';
+const DROP_INVENTORY_LIST = 'div.tw-flex-wrap.tw-tower.tw-tower--180.tw-tower--gutter-sm';
+const DROP_ITEM = '.tw-flex';
+const CATEGORY_NOT_FOUND = '[data-a-target="core-error-message"]';
+const DROP_STATUS = '[data-a-target="Drops Enabled"]';
 
 // ========================================== CONFIG SECTION =================================================================
 
+// ========================================== UTILS SECTION =================================================================
+/**
+* @fn idle
+* @param (ms) number of milliseconds to wait until the program resumes execution 
+* @author AlexSimpler
+* @return a promise which resolves after a timeout
+*/
+function idle(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
```

Comment:
What's the difference between this and `page.waitForTimeout`?

Reply:
None really  but I liked the esthetics, simpler. You could just change it to `page.waitForTimeout`